app/cookies.js

```diff
@@ -21,6 +21,11 @@ function updatePolicy (request, h, analytics) {
   cookiesPolicy.confirmed = true
 
   h.state(cookieNameCookiePolicy, cookiesPolicy)
+
+  if (!analytics) {
+    h.unstate('_ga')
+    h.unstate('_gid')
```

Comment:
I can't see a cookie with the name `_gid`, however, there are a couple of other ones:
- `_ga_TCEK75JPFK`
- `_ga_RPJ3969X9F`

I'm not sure if they are consistently named as above. I'm getting the same names locally but that doesn't mean much.
The names of the cookies are listed on the cookies page so if they are different that would also need to be updated.

Reply:
So _ga is the user tracking and _ga_123456789 is session tracking with ne user information.